packages/lab/src/skip-link/SkipLink.tsx

```diff
@@ -1,48 +1,60 @@
-import { Link, type LinkProps, makePrefixer } from "@salt-ds/core";
+import { makePrefixer } from "@salt-ds/core";
 import { useComponentCssInjection } from "@salt-ds/styles";
 import { useWindow } from "@salt-ds/window";
 import { clsx } from "clsx";
-import { type RefObject, forwardRef } from "react";
+import {
+  type ComponentPropsWithoutRef,
+  forwardRef,
+  useEffect,
+  useRef,
+} from "react";
 import { useManageFocusOnTarget } from "./internal/useManageFocusOnTarget";
 
 import skipLinkCss from "./SkipLink.css";
 
-interface SkipLinkProps extends LinkProps {
+interface SkipLinkProps extends ComponentPropsWithoutRef<"a"> {
   /**
-   * This is a ref that has access to the target element.
-   *
-   * This will be used to apply focus to that element
-   *
-   * Refs are referentially stable so if this changes it won't be picked up
-   * will need to find a better way of passing in the target element to apply the attributes
+   * The ID of the target element to apply focus when the link is clicked.
+   * If the element with this ID is not found, the SkipLink will not be rendered.
    */
-  targetRef?: RefObject<HTMLElement>;
+  target: string;
```

Comment:
```suggestion
  targetId: string;
```
Don't mind which one we choose but this is an alternative.